src/commands/participant_commands.py

```
@@ -0,0 +1,78 @@
+            return
+
+        # Send package update DM
+        from ..handlers.package_handler import send_package_update_dm
+
+        try:
+            await send_package_update_dm(bot, db, interaction.user)
+            await interaction.response.send_message(
+                "✅ Check your DMs to update package counts!", ephemeral=True
+            )
+            logger.info(f"User {interaction.user.name} requested package update")
+        except Exception as e:
```

Ruff (0.14.7)

Do not catch blind exception: `Exception`

(BLE001)